### PR TITLE
Debug logs

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**
Added DEBUG logs to RouterFS.

Tested using the Demo Spark application with `sparkContext.setLogLevel('DEBUG')`

### CODE REVIEW

File: src/main/java/io/lakefs/routerfs/PathMapper.java [hunk 1]
```diff
@@ -133,7 +133,7 @@ private List<MappingConfig> parseMappingConfig(Configuration conf) throws Invali
     /**
      * A method for troubleshooting purposes.
      */
-    private void logLoadedMappings() {
+    private void logDebugLoadedMappings() {
```
Comment (arielshaqed) [inline src/main/java/io/lakefs/routerfs/PathMapper.java:136]: We might still want the `if` protecting the contents of this file.

File: src/main/java/io/lakefs/routerfs/RouterFileSystem.java [hunk 2]
```diff
@@ -321,6 +340,11 @@ private static FileStatus switchFileStatusPathPrefix(FileStatus fileStatus, Stri
         String srcUri = mappedUri.replaceFirst(fromPrefix, toPrefix);
         Path path = new Path(srcUri);
         fileStatus.setPath(path);
+        LOG.debug("switchFileStatusPathPrefix: replaced file status with path URI \"{}\" with path URI \"{}\"", mappedUri, srcUri);
         return fileStatus;
     }
+
+    private void logTranslatedPaths(String methodName, Path srcPath, Path dstPath) {
+        LOG.debug("{}: was called with path {}, and was converted to {}", methodName, srcPath, dstPath);
```
Comment (arielshaqed) [inline src/main/java/io/lakefs/routerfs/RouterFileSystem.java]:
```suggestion
        LOG.debug("{}: path {} converted to {}", methodName, srcPath, dstPath);
```

**Review by arielshaqed — APPROVED** [review]
Neat!

I think that every per-object operation needs to be logged at level TRACE, not DEBUG.  DEBUG is fine for the loaded configuration, etc.

File: src/main/java/io/lakefs/routerfs/PathMapper.java (hunk 1 above)
Comment (Jonathan-Rosenberg) [reply src/main/java/io/lakefs/routerfs/PathMapper.java:136]: done

**Pull request merged**